### ISSUE TITLE
[Research] My VA - Payment Info: Create UAT Research Plan

### ISSUE THREAD

**Issue by hdjustice** [department-of-veterans-affairs/va.gov-team#36675]
## Background
Now that designs, development and QA is complete, we are ready to begin UAT on our new functionality.  
The first step is to create a UAT Research plan

### Resources
- [UAT research plan example](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/uat/MyVA-uat-research-plan.md)
- [UAT conversation guide example](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/2.0-redesign/uat/MyVA-uat-conversation-guide.md)
- [VSA UAT Guidance](https://github.com/department-of-veterans-affairs/va.gov-team/blob/69833737d9fe22b8990bb987e7c50de13205c5d5/platform/research/planning/what-is-uat.md)

## Tasks

- [x] Create a draft UAT Research Plan
- [x] Tag Samara for review

## Acceptance Criteria
- A research plan is created and ready to submit, once the conversation guide is ready.

**Comment by hdjustice** [comment]
@andaleliz this ticket is now in sprint 67!

**Comment by andaleliz** [comment]
Hi @Samara-Strauss! The [UAT draft](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/my-va/payment-history/uat/uat-research-plan.md) is ready for you to review. I'll start working on the conversation guide next.

**Comment by andaleliz** [comment]
I should mention that I purposely left out folks who don't receive payments, since we have access to Veterans internally (e.g. Taylor) who can verify that the section does _not_ show up for them, as expected. It doesn't feel like a good use of Perigean resources to recruit people for what would be a 5 minute check to ensure the feature wasn't showing up.

**Comment by Samara-Strauss** [comment]
Hey! This looks great! The only thing I have feedback on is the recruiting criteria. 

Generally speaking, I understand that we want to recruit diverse samples for our research so we can capture insights we may not be aware of across groups of people. However, we should do this very intentionally, and make sure that what we're asking for are things that are important to us.

> 4 participants should not have received a payment within the last 30 days.

I think this might be confusing. How can we clarify? "Someone who has received payments from the VA in the past, but has not received a payment in the last month"? 

> 75% of participants should identify as other than male (female, transgender, non-binary)

You can probably reduce this to 50%. I can't think of a testing reason that we need more female and non-binary participants than male participants. Also, someone might be transgender AND identify as male, so that's worth noting. Finally, as we learned through the profile research we just did, people are understandably cautious about sharing whether they are transgender or not, so I wouldn't call this out since we don't need to know whether someone is transgender or not for the study. 

> Secondary criteria

I would take out or clarify the following:

- **At least 6 people who use assistive technology (such as screen readers, keyboard magnification, hearing aids, or voice recognition)**: You already have a requirement for zoom magnification as a primary requirement, so this kind of confused me since they sort of overlap. I know these are nice to haves, but it is still unclear to me whether this is 6 more people using assistive technology, or an overall have of people recruited using assistive technology.
- **At least 3 transgender Veterans**: I would take this out since it doesn't have bearing on our research and we want to be sensitive to people's privacy.
- **At least 3 Veterans living abroad**: This is fine, though we'd need to accommodate whatever timezones they are in.
- **At least 2 Veterans with other than honorable discharge**: I might take this out only because I don't know if these people would qualify for benefits that receive payments. Total unknown to me, but I know that having a discharge status below honorable can limit benefits.

Let me know if you have questions!

**Comment by andaleliz** [comment]
Hi Samara! GH won't load for me so responding via email (please excuse any
formatting issues). I wanted to try to get back to you while you're still
online today.

> Hey! This looks great! The only thing I have feedback on is the recruiting
> criteria.
>
> Generally speaking, I understand that we want to recruit diverse samples
> for our research so we can capture insights we may not be aware of across
> groups of people. However, we should do this very intentionally, and make
> sure that what we're asking for are things that are important to us.
>
Yes, I totally agree. I intentionally defined the recruiting criteria to
follow-up with our previous study and try to fill some gaps we missed. For
example, we didn't talk to many women at all, so I'm trying to make up for
it with the recruiting criteria here. Across the UX practice, we've been
trying to be more inclusive with our research, and I'd love to see if
Perigean can make this work. Although I don't have a hypothesis that any
particular group would have usability problems with this feature, I think
it's important that we talk to a variety to confirm that.


> 4 participants should not have received a payment within the last 30 days.
>
> I think this might be confusing. How can we clarify? "Someone who has
> received payments from the VA in the past, but has not received a payment
> in the last month"?
>
Great point - I think your wording is more clear and I'll update when GH
comes back up for me.

> 75% of participants should identify as other than male (female,
> transgender, non-binary)
>
> You can probably reduce this to 50%. I can't think of a testing reason
> that we need more female and non-binary participants than male
> participants. Also, someone might be transgender AND identify as male, so
> that's worth noting. Finally, as we learned through the profile research we
> just did, people are understandably cautious about sharing whether they are
> transgender or not, so I wouldn't call this out since we don't need to know
> whether someone is transgender or not for the study.
>
As I mentioned above, I'd love to have this study be majority female - I'm
over estimating the needed % to account for no shows. We heard from a
mostly male group the first round of testing on this feature. I can take
out the (female, transgender, non-binary) language; I just included that to
clarify we'd be open to *anything* that wasn't male.

> Secondary criteria
>
> I would take out or clarify the following:
>
>    - *At least 6 people who use assistive technology (such as screen
>    readers, keyboard magnification, hearing aids, or voice recognition)*:
>    You already have a requirement for zoom magnification as a primary
>    requirement, so this kind of confused me since they sort of overlap. I know
>    these are nice to haves, but it is still unclear to me whether this is 6
>    more people using assistive technology, or an overall have of people
>    recruited using assistive technology.
>
> I can see how that would be confusing - I'll clarify this.

>
>    - *At least 3 transgender Veterans*: I would take this out since it
>    doesn't have bearing on our research and we want to be sensitive to
>    people's privacy.
>
> Yep, will remove this one.

>
>    - *At least 3 Veterans living abroad*: This is fine, though we'd need
>    to accommodate whatever timezones they are in.
>
> I think it's unlikely we'll get these folks, but I'll be happy to
accomodate their timezone.

>
>    - *At least 2 Veterans with other than honorable discharge*: I might
>    take this out only because I don't know if these people would qualify for
>    benefits that receive payments. Total unknown to me, but I know that having
>    a discharge status below honorable can limit benefits.
>
> I don't know either -I'm interested to see if we get anyone. I'll leave it
in, if it's all the same to you, since it's in the "nice to have" section.

Thanks for the feedback!

**Comment by Samara-Strauss** [comment]
All sounds good. Thanks, Liz! No further comments from me.

**Comment by andaleliz** [comment]
Thanks so much! I [created a ticket](https://github.com/department-of-veterans-affairs/va.gov-team/issues/37429) for @aagosto90 to finish out the details regarding scheduling, since she'll be the one running the sessions.

I also didn't hear either way whether or not we want to include co-pay questions, so I added that to her ticket as well.  Thanks for taking a look at this!
